components/sliderSection/Slider.js

```diff
@@ -184,11 +185,26 @@ const Slider = ({ sliderImages }: Props) => (
           <ChevronLeft size="large" />
         </StyledButtonBack>
         <StyledCarousel className="slider">
-          {sliderImages.map((image, index) => (
-            <StyledSlide index={index} key={image.title}>
-              <StyledImage src={`${image.url}?w=840`} alt={image.title} />
-            </StyledSlide>
-          ))}
+          <IntlConsumer>
+            {intl =>
+              sliderImages.map((image, index) => {
+                const alt = intl.translate(`sliderPhotos.${image.id}.alt`);
+                const title = intl.translate(`sliderPhotos.${image.id}.title`);
+                const defaultDesc = `Slide ${intl.translate("name")} ${index + 1}`;
+                const validate = (text: string) =>
+                  !/sliderPhotos/.test(text) ? text : defaultDesc;
```

Comment:
Writing this line as `/sliderPhotos/.test(text) ? defaultDesc : text;` would make it more immediate to understand. Negations (and especially double negations) tend to be hard to reason about.